inventories/seapath_cluster_definition_example.yml

```diff
@@ -358,3 +358,9 @@ all:
           IPv4_address 127.0.0.0/8
           IPv4_address 172.0.0.0/8
           IPv4_address 192.0.0.0/8
+
+        # Account to be created for SNMP v3 support. Optional.
+        #snmp_accounts:
+        #  - "createUser user1 SHA mypassword1 AES mypassword1"
```

Comment:
These are technically not "accounts", but commands to create accounts.
Would it be possible to add only the username and the password in the inventory, and make the playbook create the command ?

Reply:
MD5 and DES are depreciated, so we'll impose SHA and AES.